### ISSUE TITLE
Use external log folder for docker containers

### ISSUE THREAD

**Issue by pgraverdy** [icubam/icubam#288]

**Comment by rprimet** [comment]
It might be simpler to switch to the appropriate logging driver (e.g. journald) when deploying using compose and letting the host take care of that?

**Comment by dulacarnold** [comment]
Yes, that is a good strategy.  Sentry integration also helps a lot here, would be good that it be a logging mechanism that is easily manageable with GCP as well.
